### ISSUE TITLE
signs not working for players?

### ISSUE THREAD

**Issue by kelso6969** [kangarko/CoreArena#173]
[CoreArena] Enabling CoreArena v1.8.1
I have an [Arena] sign, it works for me, but not players. Even with the corearena.command.join permissions. Are there additional permissions somewhere?

**Comment by kangarko** [comment]
What happens when the user clicks on this sign? Is the sign turned into an interactive sign, i.e. has the [ma]\arena text been replaced with your sign format from settings.yml ?

**Comment by kangarko** [comment]
Yes, the spawn-protection is interfering, test with it disabled please.
